data/paradise world jobs.txt

```diff
@@ -403,3 +403,92 @@ mission "Pleasure cruise security"
 		dialog "You bid goodbye to the captain of the <npc> and accept your payment of <payment>."
 		payment
 		payment 50000
+
+
+# Brig bulk missions
+mission "Non-violent parolees to <planet>"
+	job
+	repeat
+	description `These <bunks> non-violent repeat offenders have accepted longer community service sentences that better use their skills in return for serving their time on a paradise world. Transport them to their new home to carry out their service. The payment is <payment>.`
+	passengers 20 100 .6
+	to offer
+		random < 15
+		"passenger space" > 50
+	source
+		attributes "mining" "fishing" "textiles" "factory" "farming" "oil"
+	destination
+		distance 2 18
+		attributes "paradise"
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		payment 10000
+		dialog "The passengers shuffle unhappily out of the ship under the watchful eye of a local security force as they prepare to repay their debts to society. A clerk on <origin> checks their condition as they're led off. He seems mostly pleased and hands you your payment of <payment>."
+
+
+mission "Debtors to <planet>"
```

Comment:
As mentioned elsewhere, replacements are not to be used in the unique name of the mission. Create a `name` line with this name, then give the mission a different unique name.